modules/features/podcasts/src/main/java/au/com/shiftyjelly/pocketcasts/podcasts/view/components/ratings/GiveRatingFragment.kt

```diff
@@ -87,6 +91,11 @@ class GiveRatingFragment : BaseDialogFragment() {
         }
     }
 
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+        snackBarCoordinatorLayout = (activity as? FragmentHostListener)?.snackBarView()
```

Comment:
Is it bad to keep hold of the parent activities view? The other option would be to only use it when we need it below. 
```
        (activity as? FragmentHostListener)?.snackBarView()?.let { snackBarView ->
            Snackbar.make(snackBarView, message, Snackbar.LENGTH_LONG).show()
        }
```

Reply:
Nice suggestion. I simplified the code here [88e076a](https://github.com/Automattic/pocket-casts-android/pull/2599/commits/88e076ab7a11e66cc6d1f726c9470214425e8392)